pkgs/servers/caddy/default.nix

```diff
@@ -1,16 +1,13 @@
-{ stdenv, buildGoModule, fetchFromGitHub }:
+{ stdenv, callPackage, buildGoModule, fetchFromGitHub }:
 
-buildGoModule rec {
-  pname = "caddy";
+let
+  inherit (callPackage ./common.nix { }) common utils;
+  inherit (utils) fetchFromCaddyServer;
```

Comment:
I stumbled over this - we just use it in two places, please inline the `fetchFromGitHub` in these two locations.